facial_recognition/app.py

```diff
@@ -1,7 +1,30 @@
 from flask import Flask
 
+from deepface import DeepFace
+
 app = Flask(__name__)
 
+
 @app.route("/")
 def hello_world():
-    return "<p>Hello, World!</p>"
\ No newline at end of file
+    return "<p>Team Wombat - Facial Recognition Service :) </p>"
+
+
+# TODO: Change to POST method and get the image as input
+# TODO: What if deepface fails ? - Handle error case
```

Comment:
Error case is to throw a 500 error back to the client, which initiates the manual process.